src/style.css

```diff
@@ -0,0 +1,11 @@
+body {
+  background-color: #F1EFEF;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#F1EFEF" to be "#f1efef" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693406)